packages/core/echo/echo-db/src/proxy-db/database.ts

```diff
@@ -189,9 +195,9 @@ export class EchoDatabaseImpl extends Resource implements EchoDatabase {
    */
   async loadObjectById<T = any>(
     objectId: string,
-    { timeout }: { timeout?: number } = {},
+    options: LoadObjectOptions = {},
```

Comment:
doesn't seem right that this is becoming a part of a public API. It's ok if this is just temporary to get the release out, but in that case I'd add a TODO to remove `expectedDocumentId` option

Reply:
true, moved the logic up into echo-client private method.